test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
That is the coding standard that we use at my work. Both the ? and : are typescript characters. In my mind the ? is a modifier of the :, so we require the ? and : to be together, but a space before and after them.

I was actually just thinking if we should allow
`a ? : number`
but I was hoping to leave that to someone else, if that is okay.